scripts/nlp_language_modeling/sft/preprocessing.py

```diff
@@ -80,11 +80,12 @@ def parse_conversations(tree_obj):
         raise ValueError(f'unknown role {prompt_obj["role"]}')
     turn = {'value': prompt_obj['text'], 'from': role}
     if 'labels' in prompt_obj:
-        turn['human_labels'] = prompt_obj['labels']
-        for key in turn['human_labels']:
-            value_set = label_values.get(key, set())
-            value_set.add(turn['human_labels'][key]['value'])
-            label_values[key] = value_set
+        # remove human labels
+        # turn['human_labels'] = prompt_obj['labels']
```

Comment:
why remove this? wouldnt the naming "label" get overridden and confusing after reward model annotation?

Reply:
so just looking at the data, we cant be sure if its been annotated by reward model or not? can we add a new field for it?
currently I used to match label with human_label to understand that

Reply:
```python
                    if 'label' in item['conversations'][turn]:
                        item['conversations'][turn]['gt_label'] = item['conversations'][turn]['label']
```
In the annotation, I set the human label to `gt_label`